components/CMakeLists.txt

```diff
@@ -102,25 +117,12 @@ set (CMAKE_C_FLAGS_DEBUG "")
 set (CMAKE_CXX_FLAGS_DEBUG "")
 set (CMAKE_Fortran_FLAGS_DEBUG "")
 
-if(USE_CUDA)
-  enable_language(CUDA)
-endif()
-
-# Include function definitions
-include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/cmake_util.cmake)
-include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/build_mpas_model.cmake)
-include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/build_scream.cmake)
-include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/build_model.cmake)
-
 set(BUILDCONF ${CASEROOT}/Buildconf)
 
-# Do any MPAS/scream builds first since they manage their own flags
 build_mpas_models()
-build_scream()
 
 # Set global cmake settings
 set(CMAKE_MODULE_PATH ${CIMEROOT}/src/CMake)
-set(CMAKE_VERBOSE_MAKEFILE TRUE)
```

Comment:
Thank you for removing this. :)

Reply:
Not removed, just moved farther up the file so that SCREAM builds are verbose and recorded in e3sm.bldlog like all the other components.

Reply:
Ah. Oh well. =)